src/HealthDataAIServices/HealthDataAIServices.Autorest/README.md

```diff
@@ -36,21 +36,21 @@ require:
   - $(repo)/specification/healthdataaiservices/resource-manager/readme.md
 # If the swagger has not been put in the repo, you may uncomment the following line and refer to it locally
 # - (this-folder)/relative-path-to-your-local-readme.md
-
-try-require: 
+ 
+try-require:
   - $(repo)/specification/healthdataaiservices/resource-manager/readme.powershell.md
-
+ 
 # For new RP, the version is 0.1.0
 module-version: 0.1.0
 # Normally, title is the service name
-title: HealthDeid
-subject-prefix: $(service-name)
-
+title: HealthDataAIServices
```

Comment:
Please use singular form

Reply:
`HealthDataAIServices` is the name of our resource provider, so we would like to keep it plural.